### ISSUE TITLE
NZBGET segfaults randomly

### ISSUE THREAD

**Issue by octan3** [SynoCommunity/spksrc#2407]
### Setup

_Package Name:_ Nzbget
_Package Version:_ 16.4-22

_NAS Model:_ DS415play
_NAS Architecture:_ Intel Atom
_DSM version:_ DSM 6.0.1-7393 Update 1
### Expected behavior

Nzbget runs without any interrupts.
### Actual behavior

After a bit (doing nothing) it segfaults.
### Steps to reproduce

_1._ Install nzbget.
_2._ Start nzbget.
_3._ Wait.
#### Package log

```
Tue Jul 12 20:58:47 2016        INFO    nzbget 16.4 daemon-mode
Tue Jul 12 20:59:50 2016        INFO    Reloading...
```
#### Other logs

`/var/log/messages`

```
Jul 12 21:00:55 DiskStation kernel: [82304.743366] nzbget[1239]: segfault at 0 ip b7067331 sp b43fce3c error 4 in libc.so.6[b6fe9000+1a8000]
Jul 12 21:06:04 DiskStation kernel: [82612.846658] nzbget[3222]: segfault at 0 ip b7132331 sp b46fae3c error 4 in libc.so.6[b70b4000+1a8000]

```

**Comment by kaazvaag** [comment]
I have the same issue. Nzbget randomly crashes.
After the crash I'm only able to restart it through the terminal as a root user and not by using the package center. I have upgraded from DSM 5.2 to DSM 6.0

Package Name: Nzbget
Package Version: 16.4-22

NAS Model: DS212+
NAS Architecture: 88f6281
DSM version: DSM 6.0.1-7393 Update 2

**Comment by coreequip** [comment]
Here too. But after restarting via terminal it crashes again after a couple of minutes. Can you please provide the relevant lines from you `/var/log/messages` ?

**Comment by kaazvaag** [comment]
In /var/log/messages are no lines at the time when Nzbget crashes. I do have a core dump file, but I don't have the knowledge to analyse it. Maybe Hugbug (Nzbget developer) knows how to analyse it.
I did't have any crashes on DSM 5.2 and now on DSM 6.0 it crashes randomly. So big chance it's the DSM update that's causing the crash.

**Comment by octan3** [comment]
It seems to be an incompatibility with the current DSM version. Maybe @Diaoul (maintainer of NZBGET package) can help us here?

**Comment by kaazvaag** [comment]
Downgraded to DSM 5.2. So far no more issues of sudden crashes using Nzbget. I'll keep on 5.2 till a reliable fix is made for DSM 6.0.1.

**Comment by octan3** [comment]
DSM 6.0.2-8451 - after update problem still exists.

**Comment by evilalpaca** [comment]
New user trying to install, getting this message too on DSM 6.0.2-8451. No idea about logs/dumps etc but let me know if I can provide any more info.

**Comment by octan3** [comment]
**Update**

Its still not working. I _reinstalled_ my DSM. NZBGET was not starting. I had to follow [this instructions](https://www.reddit.com/r/synology/comments/4gxk5n/cannot_install_nzbget_on_synology_with_dsm_6/d2mvkiy). Then it is running for 1 minute and I get again the segfault in the logs (here `/var/log/messages`):

    2016-09-04T16:10:26+02:00 Diskstation kernel: [ 2243.490033] nzbget[20362]: segfault at 0 ip b70cd331 sp b4624e3c error 4 in libc.so.6[b704f000+1a8000]

What to do? Segfault in `libc.so.6`? How?

**Comment by Safihre** [comment]
Closing due to age. Test package for NZBGet has been released: #3138.
We no longer compile it here on the repo, but fetch the latest official installer from NZBGet.
During package installation you can choose if you want the latest stable or testing release. After this you can use NZBGet's internal update system to do updates.
We hope to soon publish this on the official repo.
